qm-suspend: add page (#10469)
tldr-pages/tldr: pages/linux/qm-suspend.md

# qm suspend

> > Suspends a virtual machine (VM) in the Proxmox Virtual Environment (PVE).

Reviewer: ```suggestion
> Suspends a virtual machine (VM) in the Proxmox Virtual Environment (PVE).
> Use `--skiplock` and `--skiplockstorage` flags with caution, as they may lead to data corruption in certain situations.
```

> > Suspends a virtual machine (VM) in the Proxmox Virtual Environment (PVE) cluster using the "qm" command-line tool.

Reviewer 2: ```suggestion
> Suspends a virtual machine (VM) in the Proxmox Virtual Environment (PVE) .
```

> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.

> - Suspend virtual machine by id:
> 
> `qm suspend -{id} 101`

Reviewer: ```suggestion
- Suspend virtual machine by id:

`qm suspend {{vm_id}} {{integer}}`
```
We follow a similar structure for other `qm` pages.

> - To specifies the ID of the virtual machine to suspend:

Reviewer 2: ```suggestion
- Suspend virtual machine by id:
```

> `qm suspend -id 101`

Reviewer 2: id should in {}

> - To skips the lock check when suspending the VM. Use with caution, as it may lead to data corruption in certain situations:
> 
> `qm suspend --vmid 101 --skiplock`

Reviewer: ```suggestion
- Skip the lock check when suspending the VM:

`qm suspend {{vm_id}} {{integer}} --skiplock`
```

> - To skips the lock check for storage when suspending the VM. Use with caution, as it may lead to data corruption in certain situations:

Reviewer: ```suggestion
- Skip the lock check for storage when suspending the VM:
```

> `qm suspend --vmid 101 --skiplockstorage`

Reviewer: ```suggestion
`qm suspend {{vm_id}} {{integer}} --skiplockstorage`
```

# qm suspend

> Suspends a virtual machine (VM) in the Proxmox Virtual Environment (PVE).
> Use `--skiplock` and `--skiplockstorage` flags with caution, as they may lead to data corruption in certain situations.
> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.

> - Suspend virtual machine by id:

Reviewer 3: ```suggestion
- Suspend a virtual machine by id:
```
